Sources/SwiftDocC/Model/Name.swift

```diff
@@ -20,13 +20,19 @@ extension DocumentationNode {
         /// The name of a conceptual document is its title.
         case conceptual(title: String)
         /// The name of the symbol is derived from its declaration.
-        case symbol(declaration: AttributedCodeListing.Line)
+        @available(*, deprecated, message: "This deprecated API will be removed after 6.1 is released")
+        case _symbol(declaration: AttributedCodeListing.Line)
```

Comment:
Is it safe to do this with public API? I'm not familiar with the underscore naming convention here.

Reply:
Renaming the enum case is source breaking but so is adding a new case. 

The new static function allows developers to create a name value with the same code as before but any code that switches over the cases would need to be updated to handle the new `.symbol(name:)` case, regardless of how this is named.

Reply:
Now that I think about this some more. Maybe I should remove this case completely so that we don't introduce a second source breaking change when we remove this case.